### DISCUSSION TITLE
How to fold or sliding_window_view  an array?

### DISCUSSION THREAD

**Discussion by 0x1orz** [ml-explore/mlx#1207] [Q&A] [answered]
How to fold or sliding_window_view  an array ?

I attempted to reimplement the locally linear  layers like `LocallyConnected1D `,  `LocallyConnected2D` in keras.

**Comment by awni** [comment, marked as answer]
Are you looking for [`mx.as_strided`](https://ml-explore.github.io/mlx/build/html/python/_autosummary/mlx.core.as_strided.html#mlx-core-as-strided)?

**Reply by 0x1orz** [reply]
Yeah, thanks.
